design/pipeline-resources.md

```diff
@@ -27,73 +27,63 @@ resources:
 
 If you have a pipeline that produces artifacts, you can consume the artifacts by defining a `pipelines` resource. A pipeline can be another Azure DevOps pipeline or any external pipelines like Jenkins etc.
 
-
 ### Schema
 
 ```yaml
 resources:        # types: pipelines | repositories | containers | packages
   pipelines:
   - pipeline: string  # identifier for the pipeline resource
-    type: enum  # type of the pipeline source like azurePipelines, Jenkins etc. 
```

Comment:
In the schema definition, we still have to show the input `type`. For azure pipelines, this input is not required. But for other pipeline services like Jenkins this is required.

Reply:
Per an offline conversation: we'll keep around the notion of an extensible category for CI systems. It won't be "pipeline" -- that's reserved for Azure Pipelines.

Reply:
@ashokirla do you want to take a run at keywords for this, or do you want me to? I like the idea of "pipeline/upload/download" -> Azure Pipelines, with something else representing other CI/orchestration systems.

Reply:
Yeah, i am working on the proposal for the new set of keywords for external CI systems.

Reply:
i have updated the PR with the suggested changes to the spec.

Reply:
Great, resolving this conversation since you've updated the PR.